libs/partners/cohere/docs/multi_hop_agent.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "sidebar_position: 0\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cohere Tools\n",
+    "\n",
+    "The following notebook goes over how to use the Cohere tools agent:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Prerequisites for this notebook:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install package\n",
+    "!pip install -U langchain-cohere\n",
+    "%pip install wikipedia"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.agents import AgentExecutor\n",
+    "from langchain_cohere.chat_models import ChatCohere\n",
+    "from langchain_cohere.multi_hop.agent import create_cohere_multi_hop_agent\n",
+    "from langchain.retrievers import WikipediaRetriever\n",
+    "from langchain.tools.retriever import create_retriever_tool\n",
+    "from langchain_core.prompts import ChatPromptTemplate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we create the prompt template and cohere model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the prompt\n",
+    "prompt = ChatPromptTemplate.from_template(\n",
```

Comment:
Since this will act as the multihop example we should use the full multihop prompt template from libs/partners/cohere/langchain_cohere/multi_hop/prompt.py

Reply:
We use the user's prompt template and then input that into Cohere's prompt template.